src/test/java/com/microsoft/sqlserver/jdbc/spark/bulkwrite/DataSourceUtilsTest.java

```diff
@@ -11,7 +11,7 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
-package com.microsoft.sqlserver.jdbc.spark;
+package com.microsoft.sqlserver.jdbc.spark.bulkwrite;
```

Comment:
@softnado Could you please explain why you use com.microsoft.sqlserver.jdbc.spark.bulkwrite instead of previous one? I am curious :)